src/Solr/Tasks/Solr_Reindex.php

```diff
@@ -121,38 +121,4 @@ protected function doReindex($request)
         $handler->triggerReindex($this->getLogger(), $this->config()->recordsPerRequest, $taskName, $class);
     }
 
-    /**
-     * @deprecated since version 2.0.0
-     */
-    protected function runFrom($index, $class, $start, $variantstate)
```

Comment:
This code is still being referenced, you'll need to remove the reference to it as well. It looks safe to kill.

https://github.com/silverstripe/silverstripe-fulltextsearch/blob/7b72774890c37c9fa0c2499e26ec5154e31a2000/src/Solr/Tasks/Solr_Reindex.php#L93-L101

Reply:
Also line 22 that suggests you can use the `start` argument when running the task - let's kill that too